### ISSUE TITLE
Release v9.2.0 fails when TCP Router instance exists

### ISSUE THREAD

**Issue by ljfranklin** [cloudfoundry-incubator/haproxy-boshrelease#116]
Getting the following error when deploying HAProxy with a TCP Router deployed as well:

```
Task 95 | 23:51:57 | Updating instance ha_proxy: ha_proxy/e7b15db5-b8bd-421f-a604-11efa03a7086 (0) (canary) (00:07:20)
                   L Error: 'ha_proxy/e7b15db5-b8bd-421f-a604-11efa03a7086 (0)' is not running after update. Review logs for failed jobs: haproxy
Task 95 | 23:59:17 | Error: 'ha_proxy/e7b15db5-b8bd-421f-a604-11efa03a7086 (0)' is not running after update. Review logs for failed jobs: haproxy
```

The logs on the VM say:

```
[ALERT] 233/235727 (5775) : parsing [/var/vcap/jobs/haproxy/config/haproxy.config:90] : unknown option 'httpcheck'.
```

Looks like [this line](https://github.com/cloudfoundry-incubator/haproxy-boshrelease/blame/master/jobs/haproxy/templates/haproxy.config.erb#L491) says `httpcheck` when it should say `httpchk`.

**Comment by teancom** [comment]
Actually, it looks like that whole line might be wrong. I'm not sure we meant to hard-code "/health" as the URI. I'll talk to Geoff tomorrow and hopefully get this sorted out. Thanks for the report!

**Comment by geofffranks** [comment]
This issue is fixed in v9.2.1

**Comment by geofffranks** [comment]
This issue is fixed in v9.2.1
